packages/beacon-node/src/metrics/metrics/lodestar.ts

```diff
@@ -649,27 +649,44 @@ export function createLodestarMetrics(
       receivedToGossipValidate: register.histogram({
         name: "lodestar_gossip_block_received_to_gossip_validate",
         help: "Time elapsed between block received and block validated",
-        buckets: [0.05, 0.1, 0.2, 0.5, 1, 1.5, 2, 4],
+        buckets: [0.05, 0.1, 0.3, 0.5, 0.7, 1, 1.3, 1.6, 2, 2.5, 3, 3.5, 4],
```

Comment:
I'm not sure if there's any down sides specifying too many buckets like this cc @dapplion @nflaig

Reply:
The downside of specifying more buckets is that the metric becomes more expensive as it sets a value for each bucket (via `le` label). Buckets by themselves are manageable though (bounded value set), if we need this granularity to properly display it in a heatmap (or other queries) it might be ok.

What makes histogram expensive and what we need to be mostly be careful with are
- high cardinality / unbounded values in labels
- multiple labels as it creates a record for each label x value combination

In case of this metric I think we are fine but should consider this whenever we add a metric / buckets.